docs/Makefile

```diff
@@ -7,6 +7,7 @@ SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = _build
 APIDIR        = api
+GENERATEDDIR  = generated
```

Comment:
why is this a necessary addition?

Reply:
because the stuff in my folder (locally) was old and when I tried to build new docs to see how they look for `update_fits_wcsinfo` sphinx was crashing. Deleting content of this folder allows it to be re-generated if needed.

Reply:
But if it's not needed for "production" it should be removed again. I usually just clean out my local sphinx directories from time to time manually to avoid obsolete stuff.